hibernate-core/src/test/java/org/hibernate/test/annotations/access/Furniture.java

```diff
@@ -19,38 +19,6 @@
  */
 @Entity
 @AccessType("field")
-public class Furniture extends Woody {
-	@Id
-	@GeneratedValue
-	private Integer id;
+public class Furniture extends BaseFurniture {
```

Comment:
What does it have to do with this issue?

Reply:
Because we used to have an embeddable extend an entity which is wrong. This change fixes tgat as otherwise the embeddable would be register as an entity.